Makefile

```diff
@@ -16,3 +16,6 @@ pypitest: README.rst sdist
 test:
 	python -m unittest zanshinsdk/test_client.py
 
+coverage:
+	coverage run -m unittest zanshinsdk/test_client.py
+	coverage report
```

Comment:
@edpin falta uma linha no fim do arquivo

Reply:
Fixed (not that it mattered).